src/ring_theory/matrix_algebra.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Scott Morrison
 -/
 import tactic.tidy
+import tactic.squeeze
```

Comment:
This can probably be removed now, right?

Reply:
```suggestion
```

Reply:
Oops, yes. This wouldn't have been necessary in the first place if #3333 was already merged. :-)